web/src/features/panels/LeftPanel.tsx

```diff
@@ -8,7 +7,6 @@ function MapMobileHeader() {
   return (
     <div className="flex w-full items-center justify-between bg-gradient-to-b to-transparent px-2 pb-4 dark:from-black/60">
```

Comment:
Do we need the outer div here?

Reply:
Not sure but perhaps something for another day!